### PR TITLE
WIP: friends component

### PR DESCRIPTION
**Pull request by J-Rehan**

### CODE REVIEW

File: catalog/FriendScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  Alert,
+  ScrollView,
+} from 'react-native'
+import useTheme from '../components/hooks/useTheme'
+import Friend from '../components/molecules/Friend/Friend'
+import GameCard from '../components/molecules/GameCard/GameCard'
+
+const commonFriendData = {
+  imageUrl: 'https://i.imgur.com/LK7ymaN.png',
+  name: 'Lily Woofers 10',
```
Comment (krschacht) [inline catalog/FriendScreen.tsx:16]: @J-Rehan It's a small thing, but names will only ever have one space so it's more realistic for this to be "Lily Woofers10". All usernames are `[firstname] [nickname]`

**Review by krschacht — APPROVED** [review]
@J-Rehan I added a couple small comments but this PR basically looks ready to merge in

File: App.tsx [hunk 2]
```diff
@@ -5,11 +5,12 @@ import GameCardScreen from './catalog/GameCardScreen'
...
 import { useFonts } from 'expo-font'
 import ThemeContextProvider from './components/contexts/ThemeContext'
+import FriendScreen from './catalog/FriendScreen'
 
-type Views = 'ROOT' | 'GAME_CARD'
+type Views = 'ROOT' | 'GAME_CARD' | 'FRIEND'
 
 export default function App() {
-  const [view, setView] = useState<Views>('ROOT')
+  const [view, setView] = useState<Views>('FRIEND')
```
[the remark this answers is not in the document]
Comment (J-Rehan) [reply App.tsx:13]: No, It should be root. I was using`FRIEND` just for development purposes.

File: components/molecules/Friend/Friend.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+  return (
+    <View style={styles.container}>
+      <View
+        style={[
+          styles.imageContainer,
+          {
+            borderColor:
+              networkStatus === 'online'
+                ? '#0A741C'
```
[the remark this answers is not in the document]
Comment (J-Rehan) [reply components/molecules/Friend/Friend.tsx:116]: We have updated the theme in another PR #6 and once that is merged we can refactor this part.

**Pull request merged**